src/pages/institution/institution-solution/institution-solution.html

```diff
@@ -14,7 +14,7 @@ <h4 class="bold" text-wrap>{{entity?.name}}</h4>
       <ion-card *ngIf="!sol?.allowMultipleAssessemts">
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status